src/plugins/data_source/server/legacy/configure_legacy_client.ts

```diff
@@ -113,7 +142,7 @@ const getRootClient = (
  * make wrap401Errors default to false, because we don't want login pop-up from browser
  */
 const callAPI = async (
-  client: Client,
+  client: LegacyClient,
```

Comment:
Does this mean we are changing something that used the newer client to use the legacy client?